tests/src/test/scala/common/WhiskProperties.java

```diff
@@ -200,6 +200,8 @@ public static String getRouterHost() {
         return whiskProperties.getProperty("router.host");
     }
 
+    public static String getApiLocalHostName() { return whiskProperties.getProperty("whisk.api.localhost.name"); }
```

Comment:
if this is not used in open, note that you can always get a named property like this: `WhiskProperties.getProperty(name)`.